docs/utility_methods/exception_handlers.md

```diff
@@ -0,0 +1,40 @@
+# Exception handlers
+
+_Added in v0.5.7_
+
+Exceptions handlers are function that can be assigned to your app to handle exceptions that occur during the application runtime.
+They are useful for customizing the response when an error occurs, logging errors, and performing cleanup tasks.
+
+## Types
+
+Reflex support two type of exception handler `frontend_exception_handler` and `backend_exception_handler`.
+
+They are both used to handle exceptions that occur in the `frontend` and `backend` respectively.
+
+## Register an Exception Handler
+
+To register an exception handler, assign it to `app.frontend_exception_handler` or `app.backend_exception_handler` to assign a function that will handle the exception.
+
+The expected signature for an error handler is `def handler(exception: Exception)`.
+
+```md alert warning
+# Only named functions are supported as exception handler.
+```
+
+## Examples
+
+```python
+import reflex as rx
+
+def custom_frontend_handler(exception: Exception) -> None:
```

Comment:
Let's add some examples of actual handlers here to show how it could work

Reply:
What do you have in mind for actual handlers? Printing the exception ?

Reply:
Yeah I think printing it makes sense, just to show a practical way someone may use it